### DISCUSSION TITLE
Workflow artifacts with non-ascii characters in name can't be downloaded correctly

### DISCUSSION THREAD

**Discussion by Silverteal** [community/community#127672] [Actions]
### Select Topic Area

Bug

### Body

`InvalidQueryParameterValueValue for one of the query parameters specified in the request URI is invalid. RequestId:61caacf3-b01e-00c7-19b0-b8591e000000 Time:2024-06-07T07:56:48.7761753Zrscdattachment; filename="(non-ascii-chars-here)-Debug.zip"HTTP query parameter values contain invalid characters`

**Comment by grandmas-favorite** [comment]
Hi @Silverteal!

Would you be able to provide some more context about what you were trying to accomplish when you encountered this error?

**Reply by Silverteal** [reply]
Trying to download an artifact with non-ascii characters in its name. The remote server (i-forgot-subdomain.windows.net) returns error above.
